CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+# __2.0.0-preview-11__ __2018-07-30__
+## __AWS SDK for Java v2__
+  - ### Features
+    - - Updated `AwsCredentials` to interface implemented by `AwsBasicCredentials` and `AwsSessionCredentials` - Renamed `AwsCredentialsProvider.getCredentials()` to `AwsCredentialsProvider.resolveCredentials()`.
```

Comment:
Minor: Extra hypen. Formatting in markup seems a bit off due to it

Reply:
good catch, fixed